pandas/core/indexes/base.py

```diff
@@ -2788,26 +2788,55 @@ def set_value(self, arr, key, value):
 
     def _get_level_values(self, level):
         """
+        Return an Index of values for requested level.
```

Comment:
Return an Index of values for requested level. --> Return an Index of values for requested level, specially useful for MultiIndex.

It is important to make it clear somehow that this is most useful when dealing with a MultiIndex.